testconfig.yaml

```diff
@@ -1,15 +1,17 @@
 jobs:
-  - type: http
+  - type: sequence
```

Comment:
why don't use parallel as the default example?) it's more effective for ddos, especially in golang that well works with goroutines and blocking operations and better utilize cpu

Reply:
I'm working on the functionality, parallel is just there as a cherry on the pie, need to make sequence work with logic first